File: JAVA/Program/DetailFashion.java
public class DetailFashion extends KategoriProduk {
    private String idFashion, ukuran, warna, bahan;

    public DetailFashion(String idProduk, String namaProduk, int harga, int stok,
                         String idKategori, String namaKategori,
                         String idFashion, String ukuran, String warna, String bahan) {
        super(idProduk, namaProduk, harga, stok, idKategori, namaKategori);
        this.idFashion = idFashion;
        this.ukuran = ukuran;
        this.warna = warna;
        this.bahan = bahan;
    }

    @Override
    public void tampilkanInfo() {
        super.tampilkanInfo();
        System.out.println("ID Fashion    : " + idFashion);
        System.out.println("Ukuran        : " + ukuran);
        System.out.println("Warna         : " + warna);
        System.out.println("Bahan         : " + bahan);
    }
}

File: PYTHON/Program/DetailFashion.py
from KategoriProduk import KategoriProduk

class DetailFashion(KategoriProduk):
    def __init__(self, id_produk, nama_produk, harga, stok,
                 id_kategori, nama_kategori,
                 id_fashion, ukuran, warna, bahan):
        super().__init__(id_produk, nama_produk, harga, stok, id_kategori, nama_kategori)
        self.id_fashion = id_fashion
        self.ukuran = ukuran
        self.warna = warna
        self.bahan = bahan

    def tampilkan_info(self):
        super().tampilkan_info()
        print(f"ID Fashion    : {self.id_fashion}")
        print(f"Ukuran        : {self.ukuran}")
        print(f"Warna         : {self.warna}")
        print(f"Bahan         : {self.bahan}")
